repos/fdbt-site/cypress_tests/cypress/e2e/capped.cy.ts

```diff
@@ -41,4 +42,20 @@ describe('the capped faretype product journey', () => {
         completeCappedTapsJourney()
         isFinished()
     });
+
```

Comment:
The PR here is completely fine and looks right. However, we have 6 tests here (2 per type of capped journey) and I think we could just have 3 if we instead randomly decide whether to do multi-service or geoZone each time.

Could we combine these tests so we have 1 per capped journey type, and have it 50% of the time choose geoZone and 50% of the time choose multi service please?